lib/node_modules/@stdlib/stats/base/dists/kumaraswamy/examples/index.js

```diff
@@ -22,3 +22,72 @@ var objectKeys = require( '@stdlib/utils/keys' );
 var kumaraswamy = require( './../lib' );
 
 console.log( objectKeys( kumaraswamy ) );
+// Create a Kumaraswamy distribution object
```

Comment:
These changes should not be present in this PR.

Reply:
Hey @kgryte, could you clarify if #1632 should be included here? Also, please review #1632 before I move on to #2613.

Reply:
you are saying to remove #1632 right ? from this issue #2613

Reply:
Correct. A PR should be focused on one thing. You are currently mixing both #1632 and #2613. This PR should only be concerned with #2613.

Reply:
You can simply undo the Kumaraswamy changes in this PR, and this Pr can be judged on its own merits, irrespective of #1632.